module/documents/actor/actor.mjs

```diff
@@ -203,13 +203,14 @@ export default class Actor5e extends Actor {
    *                                            either a die term or a flat term.
    */
   getRollData({ deterministic=false }={}) {
-    const data = foundry.utils.deepClone(super.getRollData());
+    const data = {...super.getRollData()};
```

Comment:
I'm confused about why we need this to be a different object instead of just:
```suggestion
    const data = super.getRollData();
```

Reply:
It's as Jeff says: `Actor#getRollData` returns `this.system` and, since `deepClone` returns the original instance for complex objects, this was now returning the actor's actual `this.system` instance since it's a complex `DataModel` object now. We now copy over the properties of `this.system` since we need their derived values, but it's housed in a separate object instance so that mutations at the top-level do not mutate `this.system`. Since the object spread copies over properties and does not clone them, there's still some risk that those inner properties could be mutated, but it's difficult to find a better solution right now without support from the core software.